clients/da-vinci-client/src/main/java/com/linkedin/davinci/config/VeniceServerConfig.java

```diff
@@ -763,6 +765,7 @@ public VeniceServerConfig(VeniceProperties serverProperties, Map<String, Map<Str
         serverProperties.getInt(SERVER_DEDICATED_CONSUMER_POOL_SIZE_FOR_AA_WC_LEADER, 5);
     useDaVinciSpecificExecutionStatusForError =
         serverProperties.getBoolean(USE_DA_VINCI_SPECIFIC_EXECUTION_STATUS_FOR_ERROR, false);
+    writeBatchingPushStatus = serverProperties.getBoolean(WRITE_BATCHING_PUSH_STATUS, false);
```

Comment:
Shall we prefix this by `DaVinci`? So far, this feature seems to be DaVinci specific.

Reply:
Yup, done!